.github/workflows/data_library_open_data.yml

```diff
@@ -35,9 +40,36 @@ jobs:
           AWS_ACCESS_KEY_ID: "op://Data Engineering/DO_keys/AWS_ACCESS_KEY_ID"
 
       - uses: NYCPlanning/action-library-archive@v1.1
+        if: ${{ !(inputs.use_staging_image) }}
         with:
           name: ${{ matrix.dataset }}
           latest: true
           s3: true
           compress: true
           output_format: pgdump shapefile csv
+
+      - uses: NYCPlanning/action-library-archive@v1.2
+        if: ${{ inputs.use_staging_image }}
+        with:
+          name: ${{ matrix.dataset }}
+          latest: true
+          s3: true
+          compress: true
+          output_format: pgdump shapefile csv
+
+  create_issue_on_failure:
+    runs-on: ubuntu-22.04
+    needs: dataloading
+    if: ${{ always() && (github.event_name == 'schedule') && contains(needs.dataloading.result, 'failure') }}
```

Comment:
added the event_name clause after testing. A little tough to test it working explicitly but we use this logic elsewhere